### PR TITLE
Return JobRunsResponse for getRunsforJob method

### PR DESCRIPTION
**Pull request by ronthalanki-wework**
Updated getRunsforJob method in api/resources/JobResource to return JobRunsResponse instead of List<JobRunResponse> and updated tests accordingly

### CODE REVIEW

**Comment by ronthalanki-wework** [comment]
Closes #568

**Comment by ronthalanki-wework** [comment]
Closes #568

File: src/test/java/marquez/api/resources/JobResourceTest.java [hunk 1]
```diff
@@ -360,12 +361,9 @@ public void testGetAllRunsOfJob() throws MarquezServiceException {
     Response response =
         JOB_RESOURCE.getRunsForJob(
             NAMESPACE_NAME.getValue(), job.getName(), TEST_LIMIT, TEST_OFFSET);
+    JobRunsResponse jobRunsResponse = (JobRunsResponse) response.getEntity();
```
Comment (ashulmanWeWork) [inline src/test/java/marquez/api/resources/JobResourceTest.java]: This can be `final`

**Comment by ronthalanki-wework** [comment]
Closes #568

**Pull request merged**